### PR TITLE
remove subdomain section

### PR DESCRIPTION
**Pull request by neal-timpe**
Removed because the docs in OCP do not support this item.

Cherry pick to 4.3 and 4.4. please.

### CODE REVIEW

**Review by JStickler — APPROVED** [review]
Removing stuff? Easy peasy lemon squeezy!

LGTM.

**Comment by neal-timpe** [comment]
@yxun If you have a minute, could you just confirm that I can take this section out until the OpenShift team creates the documentation for wildcard subdomains? https://issues.redhat.com/browse/NE-278.

**Comment by yxun** [comment]
@neal-timpe I am not sure about this . Need @jwendell for a review.

File: service_mesh/service_mesh_day_two/ossm-auto-route.adoc [hunk 1]
```diff
@@ -29,40 +29,3 @@ spec:
 ----
 
 For more information, see xref:../../service_mesh/service_mesh_install/customizing-installation-ossm.adoc#ossm-cr-gateway_customizing-installation-ossm[Istio gateway configuration].
-
-[id="ossm-auto-route-subdomains_{context}"]
-== Subdomains
-
-{ProductName} creates the route with the subdomain, but {product-title} must be configured to enable it. Subdomains, for example `*.domain.com`, are supported but not by default. Cluster administrators can refer to the {product-title} documentation for instructions on how to enable subdomains.
-
-For example, if the following gateway is created:
```
Comment (jwendell) [inline service_mesh/service_mesh_day_two/ossm-auto-route.adoc:38]: The example below is not about subdomains. It is a generic example showing the expected result of using IOR. So, you just need to cut the subdomain paragraph above - and perhaps rephrase this `For example...`

**Review by jwendell — CHANGES_REQUESTED** [review]

File: service_mesh/service_mesh_day_two/ossm-auto-route.adoc (hunk 1 above)
Comment (neal-timpe) [reply service_mesh/service_mesh_day_two/ossm-auto-route.adoc:38]: You're right. Added it back, @jwendell

**Review by jwendell — APPROVED** [review]

**Review by brian-avery — APPROVED** [review]

**Review by tvieira — APPROVED** [review]
LGTM

**Pull request merged**